languages/js/src/Messages.ts

```diff
@@ -0,0 +1,12 @@
+export interface Message {
+  kind: String;
+  msg: String;
```

Comment:
```suggestion
  kind: string;
  msg: string;
```

Reply:
[TypeScript docs](https://www.typescriptlang.org/docs/handbook/basic-types.html#about-number-string-boolean-symbol-and-object):

> It can be tempting to think that the types `Number`, `String`, `Boolean`, `Symbol`, or `Object` are the same as the lowercase versions recommended above. These types do not refer to the language primitives however, and almost never should be used as a type.
>
> Instead, use the types `number`, `string`, `boolean`, `object` and `symbol`.

Reply:
word